guides/common/modules/proc_installing-webhooks-shellhooks-plugin.adoc

```diff
@@ -0,0 +1,28 @@
+[id="installing-webhooks-shellhooks-plugin_{context}"]
+= Installing the Webhooks Plugin
+
+Use the following procedure to install the webhooks plugin. 
+After installing the `foreman_webhooks` plugin, you can configure {ProjectServer} to send webhook requests.
```

Comment:
Ehm, not sure what you mean exactly. From the user perspective it will be called just `Webhooks`, but in installer (and other places) it will follow the same pattern as we do for every plugin, e.g. to install one will use `--enable-foreman-plugin-webhooks`.

Reply:
I was just wondering would they be called Satellite hooks in the downstream ;-) 
We can just refer to them as webhooks, to be safe.